frontend/src/modules/SimulationTimeSeriesSensitivity/settings.tsx

```diff
@@ -109,95 +116,91 @@ export function settings({ moduleContext, workbenchSession, workbenchServices }:
         console.debug(`handleFrequencySelectionChange()  newFreqStr=${newFreqStr}  newFreq=${newFreq}`);
         setResamplingFrequency(newFreq);
     }
-    if (!sensitivities?.getSensitivityArr()) {
-        return <div>This is not a sensitivity ensemble</div>;
+    function handleVectorSelectChange(selection: SmartNodeSelectorSelection) {
+        setSelectedVectorName(selection.selectedNodes[0]);
+    }
+    function handleShowHistorical(event: React.ChangeEvent<HTMLInputElement>) {
+        setShowHistorical(event.target.checked);
     }
 
     return (
         <>
-            <Label text="Ensemble" synced={syncHelper.isSynced(SyncSettingKey.ENSEMBLE)}>
+            <CollapsibleGroup expanded={false} title="Ensemble">
                 <SingleEnsembleSelect
                     ensembleSet={ensembleSet}
                     value={computedEnsembleIdent}
                     onChange={handleEnsembleSelectionChange}
                 />
-            </Label>
+            </CollapsibleGroup>
             <ApiStateWrapper
-                apiResult={vectorsQuery}
-                errorComponent={"Error loading vector names"}
+                apiResult={vectorsListQuery}
                 loadingComponent={<CircularProgress />}
+                errorComponent={"Could not load the vectors for selected ensembles"}
             >
-                <Label
-                    text="Vector"
-                    labelClassName={syncHelper.isSynced(SyncSettingKey.TIME_SERIES) ? "bg-indigo-700 text-white" : ""}
-                >
-                    <Select
-                        options={makeVectorOptionItems(vectorsQuery.data)}
-                        value={computedVectorName ? [computedVectorName] : []}
-                        onChange={handleVectorSelectionChange}
-                        filter={true}
-                        size={5}
-                    />
-                </Label>
+                <CollapsibleGroup expanded={true} title="Time Series">
+                    <Label text="Vector">
+                        <VectorSelector
+                            data={vectorSelectorData}
+                            selectedTags={selectedVectorName ? [selectedVectorName] : []}
```

Comment:
This should not use `selectedVectorName`. It should use a new state named `selectedVectorTag`, which is set in `handleVectorSelectChange` by `selection.selectedTags[0]`. Note that if the a default value or `fixup`-value for `selectedVectorName` is created, this must then also be set to `selectedVectorTag` state.

`selectedVectorName` is used to propagate selected vector to state for view, but `selectedTags` prop for the `VectorSelector` is a tag (e.g. `WOPT:A*`) and not a node.